integrationTests/vm/arwen/utils.go

```diff
@@ -66,7 +66,7 @@ var oneShardCoordinator = mock.NewMultiShardsCoordinatorMock(2)
 var pkConverter, _ = pubkeyConverter.NewHexPubkeyConverter(32)
 
 // GasSchedulePath --
-var GasSchedulePath = "../../../../cmd/node/config/gasSchedules/gasScheduleV2.toml"
+var GasSchedulePath = "../../../../cmd/node/config/gasSchedules/gasScheduleV4.toml"
```

Comment:
this util package.... I think we can move this var (that can be a const, actually) in the integrationTests/testInitializer.go and we can slowly dissolve this file

Reply:
fixed